Broken test on main branch: test_api_surface.py

On the main branch when running pytest this test case is failing, and causing other branches to fail the same test:

===================================================== FAILURES =====================================================
__________________________________________ test_all_test_imports_resolve ___________________________________________

    @pytest.mark.smoke
    @pytest.mark.smoke_bridge
    def test_all_test_imports_resolve():
        """Verify every supy import used in tests actually exists."""
        test_root = Path(__file__).parent
        failures = []
    
        for test_file in sorted(test_root.rglob("*.py")):
            if test_file.name in _SKIP_FILES:
                continue
            if "__pycache__" in test_file.parts:
                continue
            if test_file.name == "test_api_surface.py":
                continue
    
            try:
                tree = ast.parse(
                    test_file.read_text(encoding="utf-8", errors="replace"),
                    filename=str(test_file),
                )
            except SyntaxError:
                continue
    
            for node in ast.walk(tree):
                if isinstance(node, ast.ImportFrom):
                    if node.module and node.module.startswith("supy"):
                        _check_import_from(node, test_file, failures)
                elif isinstance(node, ast.Import):
                    for alias in node.names:
                        if alias.name.startswith("supy"):
                            _check_import(alias.name, test_file, failures)
    
        if failures:
            report = _format_report(failures)
>           pytest.fail(f"Broken supy imports in test suite:\n\n{report}")
E           Failed: Broken supy imports in test suite:
E           
E             test_init_case.py:
E               from supy import suews_sim
E                 -> supy has no attribute 'suews_sim'

test/test_api_surface.py:102: Failed

--- a/test/cmd/test_init_case.py
+++ b/test/cmd/test_init_case.py
@@ -17,7 +17,7 @@ from click.testing import CliRunner
 import pytest
 import yaml
 
-from supy import suews_sim
+import supy.suews_sim as suews_sim
 from supy.cmd.init_case import init_case_cmd
 from supy.cmd.suews_cli import cli as dispatcher
 from supy.suews_sim import SUEWSSimulation

